source/common/config/delta_subscription_impl.h

```diff
@@ -113,10 +116,10 @@ class DeltaSubscriptionImpl
 
   envoy::api::v2::DeltaDiscoveryRequest internalRequestStateForTest() const { return request_; }
 
-  // Config::SubscriptionCallbacks
-  void onConfigUpdate(const Protobuf::RepeatedPtrField<envoy::api::v2::Resource>& added_resources,
-                      const Protobuf::RepeatedPtrField<std::string>& removed_resources,
-                      const std::string& version_info) {
+  void
+  handleConfigUpdate(const Protobuf::RepeatedPtrField<envoy::api::v2::Resource>& added_resources,
```

Comment:
If this is no longer a `SubscriptCallback` (was the comment stale?), should this now be private?

Reply:
Done.